src/applications/personalization/profile-2/components/personal-information/AddressesTable.jsx

```diff
@@ -48,6 +48,7 @@ const ContactInfoCell = props => {
 const AddressesTable = ({ className, homeAddress, mailingAddress }) => (
   <ProfileInfoTable
     title="Addresses"
+    fieldName="addresses"
```

Comment:
silencing a PropType error

Reply:
okay I'll remove these changes to clean up the PR and later remove the prop. thanks!